bpy-build.yaml

```diff
@@ -2,22 +2,23 @@ addon_folder: MCprep_addon
 build_name: MCprep_addon
 
 install_versions:
-  - '4.0'
-  - '3.6'
-  - '3.5'
-  - '3.4'
-  - '3.3'
-  - '3.2'
-  - '3.1'
-  - '3.0'
-  - '2.93'
-  - '2.90'
-  - '2.80'
+  - 4.0
+  - 3.6 
+  - 3.5
+  - 3.4
+  - 3.3
+  - 3.2
+  - 3.1
+  - 3.0
+  - 2.93
+  - 2.9
+  - 2.83
+  - 2.8
 
-# Run using a command like: bpy-addon-build --during-build dev
-during_build:
-  default:
-    - [] # Re-enable later: [flake8 --extend-ignore W191 .]
-  dev:
-    - create_file("mcprep_dev.txt")
-  
+build_actions:
+  default: 
+    script: "action-scripts/default.py"
+  dev: 
+    script: "action-scripts/dev.py"
+    ignore_filters: 
+      - "*.blend"
```

Comment:
On further review, I don't think this filter works very well because it just results in the blend files being missing in the final archive right?

I was most of the way through implementing a "fast" build option which set the ignore filter to be the mcprep_resources folder, and that seemed to be working fine until I realize we fully delete the existing install always via the line:

```
if addon_path.exists():
    shutil.rmtree(addon_path)
```

Not having any of the source files means test fail, so not very useful for dev building. I'd remove this until we have a proper fast flag that can leave some files in place.